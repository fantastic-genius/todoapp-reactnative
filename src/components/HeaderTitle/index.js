import React from 'react';
import { View, Text, StyleSheet } from 'react-native';

const HeaderTitle = ({ title }) => {
  return (
    <View
      style={styles.contanier}
    >
      <Text style={styles.title}>{title}</Text>
    </View>
  );
};

const styles = StyleSheet.create({
  contanier: {
    alignItems: 'center',
    justifyContent: 'center',
    flex: 1
  },
  title: {
    color: '#d3d7de',
    fontSize: 25,
    lineHeight: 50
  }
})

export default HeaderTitle;
