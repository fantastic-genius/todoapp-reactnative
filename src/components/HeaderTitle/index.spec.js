import React from 'react';
import renderer from 'react-test-renderer';

import HeaderTitle from './index';

describe('<HeaderTitle />', () => {
  it('Has one child', () => {
    const component = renderer.create(<HeaderTitle />).toJSON();
    expect(component.children.length).toBe(1);
  });

  it('renders correctly', async () => {
    const component = renderer.create(<HeaderTitle />).toJSON();
    expect(component).toMatchSnapshot();
  });
});

