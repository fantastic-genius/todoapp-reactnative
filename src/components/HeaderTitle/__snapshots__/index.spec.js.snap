// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HeaderTitle /> renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text
    style={
      Object {
        "color": "#d3d7de",
        "fontSize": 25,
        "lineHeight": 50,
      }
    }
  />
</View>
`;
